apis/management.cattle.io/v3/rke_types.go

```diff
@@ -552,3 +556,11 @@ type AzureCloudProvider struct {
 // AWSCloudProvider options
 type AWSCloudProvider struct {
 }
+
+// MetricsServerConfig
+type MetricsServerConfig struct {
+	// Disable metrics server
+	Disable bool `yaml:"disable" json:"disable,omitempty"`
```

Comment:
can we just use None value to disable it? again, similar to other addons

Reply:
and whatever provider is used for the current metric server as an alternative

Reply:
the problem is that metrics server doesn't have plugins like ingress, but it does contain options